library/ui/src/main/res/layout/exo_playback_control_view.xml

```diff
@@ -31,30 +31,39 @@
     android:orientation="horizontal">
 
     <ImageButton android:id="@id/exo_prev"
+      android:contentDescription="@string/exo_controls_previous_description"
```

Comment:
These are already added via the styles.

Reply:
Right. Good to know - thanks. This came from Android Studio's linter....I will file a bug. 

![image](https://user-images.githubusercontent.com/873212/66751979-3d204a00-ee88-11e9-85ef-9fe07d4fc937.png)